src/components/CustomerDetails/CustomerDetails.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import CustomerDetails from '../CustomerDetails';
+
+describe('CustomerDetails', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(
+        <CustomerDetails/>
+      );
+  });
+
+  it('renders a customer form first heading ', () => {
+    expect(wrapper.find("h3").at(0).text()).toEqual("Shipping Details");
+  });
+
+  it('renders a customer form fourth label ', () => {
+    expect(wrapper.find("FormItem").at(3).text()).toEqual("City");
+  });
+
+  it('renders a customer form second step', () => {
+    expect(wrapper.find("Step").at(1).text()).toEqual("Items")
+  })
+
+  it('contains two postcodes for shipping + billing', () => {
+    expect(wrapper.find('[label="Postcode"]').length).toEqual(2);
```

Comment:
Regarding brittleness of tests: what happens when we internationalise these labels? Do we want to couple our tests to specific translations

Reply:
Would the javascript version of symbols be a good replacement? Or do they not behave in the same way?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol
e.g.
```javascript
Symbol('postcode')
```

Reply:
Symbols aren't really a thing in the JS world.